modules/mono/editor/GodotTools/GodotTools.Shared/GodotTools.Shared.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+    </PropertyGroup>
+
```

Comment:
```suggestion
```

Also, I noticed here the target is `netstandard2.0`, while Godot projects generate with `net472`, why the inconsistency? Also, there is `netstandard2.1`.

Reply:
Godot projects didn't support .NET Standard at first. Today we still default to `net472` in 3.2 to avoid breaking projects, but 3.2 supports it and 4.0 defaults to `netstandard2.1`.
Regarding `GodotTools`, .NET Standard is definitely preferred, but some projects have .NET Framework dependencies like `Microsoft.Build` hence why they target .NET Framework instead (.NET Standard projects can't have .NET Framework dependencies).
The reason we target `netstandard2.0` rather than `netstandard2.1` is because the latter is only fully supported in `net480` (.NET Framework 4.8) which our projects don't target as it was too bleeding edge back then.
Also .NET Standard 2.0 is recommended over 2.1 for the best compatibility (as said before `net472` can't reference `netstandard2.1`) so `GodotTools.IdeMessaging` likely won't change as it's used outside of Godot too (VS extension).